### PR TITLE
HtmlView component

### PR DESCRIPTION
**Pull request by enchev**

### CODE REVIEW

**Comment by hdeshev** [comment]
:+1: 

hipster

**Comment by atanasovg** [comment]
:100: Great!

**Comment by vakrilov** [comment]
:+1:

File: apps/tests/ui/html-view/html-view-tests.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+// </snippet>
+
+// ### Declaring a HtmlView.
+//```XML
+//  <Page>
+//      <HtmlView html="{{ htmlString }}" />
+//  </Page>
+//```
+
+// </snippet>
```
Comment (vakrilov) [inline apps/tests/ui/html-view/html-view-tests.ts]: just a closing snippet tag ?

File: apps/tests/ui/html-view/html-view-tests.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    var htmlView = _createHtmlViewFunc();
+    var pageFactory = function (): page.Page {
+        newPage = new page.Page();
+        newPage.content = htmlView;
+        return newPage;
+    };
+
+    helper.navigate(pageFactory);
+
+    // <snippet module="ui/html-view" title="HtmlView">
```
Comment (vakrilov) [inline apps/tests/ui/html-view/html-view-tests.ts]: just an opening snippet tag ?

File: apps/tests/ui/html-view/html-view-tests.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+// Using a HtmlView requires the html-view module.
+// ``` JavaScript
+import htmlViewModule = require("ui/html-view");
+// ```
+// </snippet>
+
+// ### Declaring a HtmlView.
+//```XML
+//  <Page>
+//      <HtmlView html="{{ htmlString }}" />
```
Comment (vakrilov) [inline apps/tests/ui/html-view/html-view-tests.ts]: Wrap with {%raw%}   {%endraw%} tags because of the binding

**Pull request merged**